manifests/s/shirowanisan/COEIROINK/0.2.0/shirowanisan.COEIROINK.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: shirowanisan.COEIROINK
+PackageVersion: 0.2.0
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: COEIROINK-CPU-v.0.2.0/torch/bin/protoc.exe
+UpgradeBehavior: install
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 848.)